src/sources/aws_s3/sqs.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(unused)]
```

Comment:
Is this still necessary? I think we would want to avoid that kind of blanket allow.

Reply:
removed

Reply:
Resolved but unchanged?